### DISCUSSION TITLE
作用域里找不到android.miui.overlay

### DISCUSSION THREAD

**Discussion by ZLHuo** [LSPosed/LSPosed#2541] [Q&A] [answered]
如题，系统为MIUI14，该应用位于/product/overlay/MiuiFrameworkResOverlay.apk，包名为android.miui.overlay，lsposed中即使取消所有隐藏也找不到该应用，求解

**Comment by Howard20181** [comment, marked as answer]
hook overlay 没有用，你要hook overlay覆盖的那个目标应用

**Reply by ZLHuo** [reply]
好的，我再研究一下，非常感谢！
